### PR TITLE
add scheduler integration benchmark

### PR DESCRIPTION
**Pull request by lavalamp**
Ref #14216

### CODE REVIEW

**Comment by lavalamp** [comment]
@davidopp

**Comment by ixdy** [comment]
this one is assigned to me, but I'm entirely unfamiliar with the integration test, so someone else would probably be a better reviewer.

File: test/integration/scheduler_test.go [hunk 1]
```diff
@@ -291,3 +292,149 @@ func DoTestUnschedulableNodes(t *testing.T, restClient *client.Client, nodeStore
...
+		b.Fatalf("Couldn't create scheduler config: %v", err)
+	}
+	eventBroadcaster := record.NewBroadcaster()
+	schedulerConfig.Recorder = eventBroadcaster.NewRecorder(api.EventSource{Component: "scheduler"})
+	eventBroadcaster.StartRecordingToSink(c.Events(""))
+	scheduler.New(schedulerConfig).Run()
+
+	defer close(schedulerConfig.StopEverything)
+
+	makeNNodes(c, 1000)
```
Comment (davidopp) [inline test/integration/scheduler_test.go]: Is it a good idea for the scheduler and the test to use the same client? I guess it works...

**Comment by davidopp** [comment]
LGTM

**Pull request merged**